src/handler.js

```diff
@@ -46,48 +47,54 @@ const createProxy = config => {
   return proxy;
 };
 
-module.exports = config => (req, res) => {
-  const proxy = createProxy(config);
-
-  const method =
-    req.method && req.method.toUpperCase && req.method.toUpperCase();
+module.exports = options => {
+  const config = getConfig(options);
+  const proxy = createProxy(config.airtableApiKey);
```

Comment:
This was an unrelated but important fix. With the previous code, we'd be creating a new proxy on every new request, which isn't very efficient. Sorry, I should've caught this in past code review.